bedrock/firefox/templates/firefox/features/adblocker.html

```diff
@@ -0,0 +1,216 @@
+ {% endblock %}
+
+ {% block page_css %}
+   {{ super() }}
+   {{ css_bundle('adblocker') }}
+ {% endblock %}
+
+ {% macro card_block(class, tag_label, media_icon, title, ga_title, desc, image_url, include_highres_image, video_url, aspect_ratio, link_url, meta, cta) -%}
+
+ <section class="mzp-c-card {% if class %}{{ class }}{% endif %} {% if aspect_ratio %}{{ aspect_ratio }}{% endif %} {% if media_icon %}{{ media_icon }}{% endif %} {% if video_url %}has-video-embed{% endif %}">
+   <a class="mzp-c-card-block-link" href="{{ link_url }}" data-link-name="{{ ga_title }}" data-link-type="link">
+     <div class="mzp-c-card-media-wrapper">
+       {{ lazy_img(image_url=image_url, include_highres_image=include_highres_image, placeholder_url='placeholder.png', optional_attributes={'class': 'mzp-c-card-image'}) }}
+     </div>
+     <div class="mzp-c-card-content">
+     {% if tag_label %}
+       <div class="mzp-c-card-tag {% if tag_class %}{{ tag_class }}{% endif %}">{{ tag_label }}</div>
+     {% endif %}
+       <h2 class="mzp-c-card-title"><span>{{ title }}</span></h2>
+     {% if desc %}
+       <p class="mzp-c-card-desc">{{ desc }}</p>
+     {% endif %}
+     {% if meta %}
+       <p class="mzp-c-card-meta">{{ meta }}</p>
+     {% endif %}
+     {% if cta %}
+       <p class="mzp-c-card-cta"><span class="mzp-c-card-cta-text">{{ cta }}</span></p>
+     {% endif %}
+     </div>
+   </a>
+   {% if video_url %}
+   <div class="mzp-c-card-video-wrapper hidden">
+     <figure class="mzp-c-card-video-content">
+       <video class="mzp-c-card-video" preload="none" controls playsinline data-ga-label="{{ ga_title }}">
+         <source src="{{ video_url }}.webm" type="video/webm">
+         <source src="{{ video_url }}.mp4" type="video/mp4">
+       </video>
+       <figcaption>
+         <p>{% if desc %}{{ desc }}{% endif %} <a href="{{ link_url }}">{{ _('Read more') }}</a></p>
+       </figcaption>
+     </figure>
+   </div>
+   {% endif %}
+ </section>
+
+ {%- endmacro %}
+
+ {% block content %}
+
+  <section class="mzp-c-hero mzp-t-product-firefox">
+    <div class="mzp-l-content">
+      <div class="mzp-c-hero-body">
+        <h1 class="mzp-c-hero-title">{{ _('Everything you need to know about ad blockers.') }}</h1>
+        <div class="mzp-c-hero-desc">
+          <p>{{ _('Ad blocker is a web browser extension-- that can detect web ads and prevent them from being displayed on a web page.') }}</p>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="mzp-c-card-feature mzp-has-aspect-3-2 mzp-l-card-feature">
+    <div class="mzp-c-card-feature-media-wrapper">
+      {{ high_res_img('firefox/features/privacy-illustration1.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
+    </div>
+    <h1 class="mzp-c-hero-title">{{ _('How ad blockers work') }}</h1>
+    <div class="mzp-c-card-feature-content">
+      <div class="mzp-l-content mzp-l-card-half">
+        <div class="mzp-c-card">
```

Comment:
Hmm, I don't think we embedding a card component inside a feature card is something we've done before. I don't think there's anything wrong with that per-say, but this content doesn't look quite like a card to me (it doesn't link anywhere etc). I think this would be fine just as a regular heading with copy?